versioningMatrix.json

```diff
@@ -1,30 +1,30 @@
 {
-    "2.1.6": {
-        "sdk": ">=1016.0.0 <1018.0.0",
-        "api": ">=1016.0.0 <1018.0.0"
-    },
-    "3.0.0": {
-        "sdk": ">=1018.0.0",
-        "api": ">=1018.0.0"
-    },
-    "3.0.1": {
-        "sdk": ">=1018.0.0",
-        "api": ">=1018.0.0"
-    },
-    "3.1.0": {
-        "sdk": ">=1018.0.0",
-        "api": ">=1018.0.0"
-    },
-    "3.2.0": {
-      "sdk": ">=1018.0.0",
-      "api": ">=1018.0.0"
-    },
-    "3.2.1": {
-      "sdk": ">=1018.0.0",
-      "api": ">=1018.0.0"
-    },
-    "3.2.2": {
-      "sdk": ">=1018.0.0",
-      "api": ">=1018.0.0"
-    }
+  "2.1.6": {
+    "sdk": ">=1016.0.0 <1018.0.0",
+    "api": ">=1016.0.0 <1018.0.0"
```

Comment:
So 1018 is never supported?

Reply:
3.0.0 is the version in which we bumped @c8y components from 1016 to 1020, so all versions after are indeed not compatible. We have a few versions before 2.1.6 which are not listed in the matrix, but I assume there was a reason for this...